src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -867,7 +867,7 @@ private static String processPageSourceForGodMode(String content) {
                 // now (used in comments last edited date) e.g. [Thu, 07 May 2015, 07:52:13 UTC]
                 .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC", "\\${comment\\.date}")
                 // now (used in opening time/closing time Grace period)
-                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}", "\\${grace\\.period\\.date}")
+                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}", "\\${datetime\\.now}")
```

Comment:
Hmmm why is the `[AP]M` part of the regex removed? Won't it cause incorrect regeneration?
Also, the comment above this `replaceAll` part needs to be updated.